.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
```

Comment:
Now this is the super simple workflow that will run on every PR targeting `master` or `dev`